ostd/src/arch/x86/tdx_guest.rs

```diff
@@ -68,15 +73,21 @@ pub unsafe fn unprotect_gpa_range(gpa: Paddr, page_num: usize) -> Result<(), Pag
     .map_err(|_| PageConvertError::TdVmcall)
 }
 
-/// Sets the given physical address range to Intel TDX private pages.
-/// Make sure the provided physical address is page size aligned.
+/// Converts physical pages to Intel TDX private pages.
 ///
-/// # Safety
+/// This function clears the [`PrivFlags::SHARED`] bit in the linear mapping of physical pages.
+/// Then, it invokes the [`map_gpa`] and [`accept_page`] TDVMCALLs to convert those pages into
```

Comment:
In fact, there is a difference between TDCALL and (TD)VMCALL. TDCALL is used to call guest-side Intel TDX functions. TDVMCALL is a specific form of TDCALL, which invokes a host VM service. The `map_gpa` belongs to TDVMCALL, while the `accept_page` belongs to TDCALL.

Reply:
Oh, sorry. I'm a bit careless and didn't catch the differences in your original comment. Your detailed explanation helped me understand! [Addressed now.](https://github.com/asterinas/asterinas/compare/1d11e0e0c30b6083596ec15db0e925bdac832562..0e4bb15e4fb1b88c12c73f00c21724abb488471b)